aztec/src/main/kotlin/org/wordpress/aztec/util/DynamicLayoutGetBlockIndexOutOfBoundsException.kt

```diff
@@ -0,0 +1,5 @@
+package org.wordpress.aztec.util
```

Comment:
What do you think about moving this class to a package of its own like `org.wordpress.aztec.exception` or `org.wordpress.aztec.util.exception`?